packages/openlayers/lib/map/create-map.ts

```diff
@@ -33,6 +36,32 @@ export function createLayer(layerModel: MapContextLayer): Layer {
         }),
       });
       break;
+    case "ogcapi":
+      if (layerModel.layerType === 'vectorTiles') {
```

Comment:
Here we might use ogc-client to figure out the right urls for each type of layer, instead of expecting the user to provide it